src/PrestaShopBundle/Event/Dispatcher/NullDispatcher.php

```diff
@@ -58,4 +58,8 @@ public function removeListener($eventName, $listener)
     public function removeSubscriber(EventSubscriberInterface $subscriber)
     {
     }
+
+    public function getListenerPriority($eventName, $listener)
```

Comment:
?

Reply:
you need to implement the interface which have one more method ;)